developer-docs-site/docs/guides/prover-guides/prover-guide.md

```diff
@@ -150,10 +146,7 @@ error: abort not covered by any of the `aborts_if` clauses
 }
 ```
 
-The prover has generated a counter example which leads to an overflow when adding 1 the value of 255 for an `u8`. This
-happens if the function specification states something abort behavior, but the condition under which the function
-is aborting is not covered by the specification. And in fact, with `aborts_if !exists<Counter>(a)` we only cover the
-abort if the resource does not exists, but not the overflow.
+The Move Prover has generated an example counter that leads to an overflow when adding 1 to the value of 255 for an `u8`. This overflow occurs if the function specification calls for abort behavior, but the condition under which the function is aborting is not covered by the specification. And in fact, with `aborts_if !exists<Counter>(a)` we only cover the abort if the resource does not exist, but not the overflow.
```

Comment:
Please clarify: “And in fact, with `aborts_if !exists<Counter>(a)` we only cover the abort if the resource does not exist, but not the overflow.”

Are we saying, “And in fact, with `aborts_if !exists<Counter>(a)` we trigger the abort only if the resource does not exist but do not trigger the abort on overflow” here?

Reply:
Revised.